### PR TITLE
 refactor: update stack with component-rollup-config

### PR DESCRIPTION
**Pull request by antleblanc**
# Update stack

### 🗜 Refactor

71791ce — refactor: update stack with component-rollup-config

### 💥 Breaking Change

Module is now named as `ngOvhFormFlat`

ref: ovh-ux/manager#192

### CODE REVIEW

File: src/flat-checkbox/index.less [hunk 1]
```diff
@@ -1,6 +1,5 @@
 @flat-checkbox-width: 18px;
```
Comment (cbourgois) [inline src/flat-checkbox/index.less:1]: Should be scoped : 

```less 
flat-checkbox, [data-flat-checkbox], [flat-checkbox]
```

File: src/flat-radio/index.less [hunk 2]
```diff
@@ -1,25 +1,27 @@
 @flat-radio-width: 17px;
 @flat-radio-height: @flat-radio-width;
-
 @flat-radio-border-color: #8d919c;
-@flat-radio-color: #fff;
 @flat-radio-focus-border-color: #009fe3;
 @flat-radio-disabled-border-color: #ddd;
 
 .flat-radio-input-hover {
```
Comment (cbourgois) [inline src/flat-radio/index.less:7]: Should be scoped : 

```less 
flat-radio, [data-flat-radio], [flat-radio]
```

File: src/flat-select/index.less [hunk 3]
```diff
@@ -0,0 +1,66 @@
+@form-flat-select-default: #8d919c;
+@form-flat-select-focus: #2574A9;
+@form-flat-select-focus-background: #b9e4f7;
+
+div.flat-select {
```
Comment (cbourgois) [inline src/flat-select/index.less:5]: Should be scoped : 

```less 
flat-select, [data-flat-select], [flat-select]
```

File: src/flat-switch/index.less [hunk 4]
```diff
@@ -1,29 +1,26 @@
...
+// Cursor
+@flat-switch-cursor-size: 28px;
+@flat-switch-cursor-width: @flat-switch-cursor-size;
+@flat-switch-cursor-height: @flat-switch-cursor-size;
+@flat-switch-cursor-color: #FFF;
+@flat-switch-cursor-color: hsl(168, 32%, 54%); //#65afa0;
+@flat-switch-cursor-border-color: #53a191;
+@flat-switch-cursor-radius: 50%;
 
 .flat-switch {
```
Comment (cbourgois) [inline src/flat-switch/index.less:19]: Should be scoped : 

```less 
flat-switch, [data-flat-switch], [flat-switch]
```

**Review by cbourgois — CHANGES_REQUESTED** [review]

File: src/flat-checkbox/index.less (hunk 1 above)
Comment (antleblanc) [reply src/flat-checkbox/index.less:1]: Variables are used across all `.*less` files.

Styles are already scoped by `.flat-*`.

Your suggestions doesn't work. Did I miss something? Thanks.

File: src/flat-radio/index.less (hunk 2 above)
Comment (antleblanc) [reply src/flat-radio/index.less:7]: Please refer to my previous comment.

File: src/flat-select/index.less (hunk 3 above)
Comment (antleblanc) [reply src/flat-select/index.less:5]: Please refer to my previous comment.

File: src/flat-switch/index.less (hunk 4 above)
Comment (antleblanc) [reply src/flat-switch/index.less:19]: Please refer to my previous comment.

File: src/flat-checkbox/index.less (hunk 1 above)
Comment (antleblanc) [reply src/flat-checkbox/index.less:1]: These selectors can be reached due to the `replace: true`.

Comment (cbourgois) [reply src/flat-checkbox/index.less:1]: Seems legit.

**Review by cbourgois — APPROVED** [review]

**Review by FredericEspiau — APPROVED** [review]

**Pull request merged**